brix/functions.py

```diff
@@ -3,3 +3,45 @@
 from .classes import Handler
 from .classes import Indicator
 from .classes import CompositeIndicator
+
+import requests
+
+def update_geogrid_data(H, update_func, geogrid_data=None, **kwargs):
+  '''
+  High order function to update table geogrid data. Based on `update_package`
+   and `perform_update` methods from `Handler` class.
+
+  NOTE: From @crisjf "Eventually could be added as a method of the Handler"
+  WARNING: From @RELNO: "Edits to the grid from a different source might not be
```

Comment:
When we add this functionality to Handler, we'll make sure to check for inconsistencies before posting. Mainly, we need to make sure not to post a cell type not defined when the table was created.